File: src/main/php/Easy/ArrayPairSum.php
<?php

declare(strict_types=1);

namespace Src\Easy;

class ArrayPairSum
{
    public function solve(array $nums): int
    {
        $total = 0;
        sort($nums);
        for ($i = count($nums) - 1; $i >= 0; $i -= 2) {
            $total += min($nums[$i], $nums[$i - 1]);
        }

        return $total;
    }
}

File: src/main/java/easy/ArrayPairSum.java
package easy;

import java.util.Arrays;

public class ArrayPairSum {
    public int solve(int[] nums) {
        int sum = 0;
        Arrays.sort(nums);
        for (int i = nums.length - 2; i >= 0; i -= 2) {
            sum += Math.min(nums[i], nums[i + 1]);
        }

        return sum;
    }
}
